plans/install.pp

```diff
@@ -45,7 +45,7 @@
 
   # Common Configuration
   String                            $console_password,
-  Peadm::Pe_version                 $version                          = '2021.7.4',
+  Peadm::Pe_version                 $version,
```

Comment:
why do you remove it?

Reply:
I'm wondering if this is a breaking change, I'm not sure. Why did product decide this? Is the discussion public somewhere?

Reply:
Afternoon Tim, I reiterated your concern with product and we're going to keep default as latest LTS.